 * Serving Flask app 'app'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.31.92.215:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 433-626-050
120.56.38.160 - - [04/Aug/2024 07:05:40] "GET / HTTP/1.1" 200 -
120.56.38.160 - - [04/Aug/2024 07:05:45] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
120.56.38.160 - - [04/Aug/2024 07:06:52] "GET / HTTP/1.1" 200 -
120.56.38.160 - - [04/Aug/2024 07:06:53] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
103.82.211.234 - - [04/Aug/2024 07:07:40] "GET / HTTP/1.1" 200 -
103.82.211.234 - - [04/Aug/2024 07:07:45] "[32mPOST /login HTTP/1.1[0m" 302 -
103.82.211.234 - - [04/Aug/2024 07:07:46] "GET /videos HTTP/1.1" 200 -
103.82.211.234 - - [04/Aug/2024 07:08:32] "GET / HTTP/1.1" 200 -
103.82.211.234 - - [04/Aug/2024 07:08:39] "[32mPOST /login HTTP/1.1[0m" 302 -
103.82.211.234 - - [04/Aug/2024 07:08:40] "GET /videos HTTP/1.1" 200 -
103.82.211.234 - - [04/Aug/2024 07:08:46] "GET /documents HTTP/1.1" 200 -
120.56.38.160 - - [04/Aug/2024 07:10:05] "POST /login HTTP/1.1" 200 -
120.56.38.160 - - [04/Aug/2024 07:10:13] "[32mPOST /login HTTP/1.1[0m" 302 -
120.56.38.160 - - [04/Aug/2024 07:10:13] "GET /videos HTTP/1.1" 200 -
120.56.38.160 - - [04/Aug/2024 07:10:19] "GET /documents HTTP/1.1" 200 -
103.82.211.234 - - [04/Aug/2024 07:59:37] "GET /documents HTTP/1.1" 200 -
118.123.105.93 - - [04/Aug/2024 08:40:54] "GET / HTTP/1.1" 200 -
198.235.24.27 - - [04/Aug/2024 09:29:51] "GET / HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:12:55] "GET / HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:12:57] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
106.206.56.78 - - [04/Aug/2024 10:13:26] "POST /login HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:14:02] "GET /register HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:15:15] "GET / HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:15:15] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
49.44.78.4 - - [04/Aug/2024 10:15:16] "GET / HTTP/1.1" 200 -
106.206.56.78 - - [04/Aug/2024 10:15:31] "[32mPOST /login HTTP/1.1[0m" 302 -
106.206.56.78 - - [04/Aug/2024 10:15:31] "GET /videos HTTP/1.1" 200 -
49.44.86.229 - - [04/Aug/2024 10:15:32] "[31m[1mGET /login HTTP/1.1[0m" 405 -
106.206.56.78 - - [04/Aug/2024 10:15:56] "GET /documents HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:20:33] code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00#\x00\x00\x00')
152.58.253.251 - - [04/Aug/2024 10:20:33] "[35m[1m\x16\x03\x01\x06Ã€\x01\x00\x06Â¼\x03\x03&\x9e\x0cBÃ„ÃƒÂ»~cÂª*gÂ¶\x8ekÃºÂ¾) \x1cC2\x8fÃ£\x9d\x01s~Ã‹\x85x\x1f Ãš\x9bÃ°\x05~c"\x8c{nÃ”/5Ã…\x83Ãž)\x07ÂµÃ„Âª%Ã¦o#\x8a{Ã’Â¡Ã“Â­\x88\x00 ÂªÂª\x13\x01\x13\x02\x13\x03Ã€+Ã€/Ã€,Ã€0ÃŒÂ©ÃŒÂ¨Ã€\x13Ã€\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06S\x8a\x8a\x00\x00\x00\x0d\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00#\x00\x00\x00[0m" HTTPStatus.BAD_REQUEST -
152.58.253.251 - - [04/Aug/2024 10:20:33] code 400, message Bad request version ('\x8a\x8a\x13\x01\x13\x02\x13\x03Ã€+Ã€/Ã€,Ã€0ÃŒÂ©ÃŒÂ¨Ã€\x13Ã€\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06\x93ÃªÃª\x00\x00\x00')
152.58.253.251 - - [04/Aug/2024 10:20:33] "[35m[1m\x16\x03\x01\x07\x00\x01\x00\x06Ã¼\x03\x03*<\x86\x8dPÃ¼NDR\x91Ã¨\x9eÃŒ\x1a`Ã«Ã¤>Â²Â¨\x84\x12\x8cÃŽ\x89\x9e\x9d\x0bB\x92y\x8c \x99Ã§Ã¥=GÃ˜k3bK>$C)n:(F"r\x1b\x05Â¤JÃª\x9bÃ¢*\x9e\x0c\x1d\x17\x00 \x8a\x8a\x13\x01\x13\x02\x13\x03Ã€+Ã€/Ã€,Ã€0ÃŒÂ©ÃŒÂ¨Ã€\x13Ã€\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06\x93ÃªÃª\x00\x00\x00[0m" HTTPStatus.BAD_REQUEST -
152.58.253.251 - - [04/Aug/2024 10:20:37] "GET / HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:20:41] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
152.58.253.251 - - [04/Aug/2024 10:21:01] "POST /login HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:21:10] "POST /login HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:21:15] "GET / HTTP/1.1" 200 -
49.44.67.138 - - [04/Aug/2024 10:21:33] "GET / HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:21:36] "GET / HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:21:58] "[32mPOST /login HTTP/1.1[0m" 302 -
152.58.253.251 - - [04/Aug/2024 10:21:58] "GET /videos HTTP/1.1" 200 -
152.58.253.251 - - [04/Aug/2024 10:22:06] "GET /documents HTTP/1.1" 200 -
45.156.130.4 - - [04/Aug/2024 12:51:37] "GET / HTTP/1.1" 200 -
31.220.96.113 - - [04/Aug/2024 14:31:44] "GET / HTTP/1.1" 200 -
134.209.0.28 - - [04/Aug/2024 16:49:35] "GET / HTTP/1.1" 200 -
205.210.31.70 - - [04/Aug/2024 18:47:40] "GET / HTTP/1.1" 200 -
138.68.174.63 - - [04/Aug/2024 19:17:15] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03Ã²)PÃ”Ã‘\x81\x11Â¦fÃ—ÃˆÃ£Â½=Ã©M\x8fcÃ¼\x18\x10Ãˆ\x9a\x98Ã™|Âº\x965|\xadÃŽ\x00\x00\x1aÃ€/Ã€+Ã€\x11Ã€\x07Ã€\x13Ã€')
138.68.174.63 - - [04/Aug/2024 19:17:15] "[35m[1m\x16\x03\x01\x00{\x01\x00\x00w\x03\x03Ã²)PÃ”Ã‘\x81\x11Â¦fÃ—ÃˆÃ£Â½=Ã©M\x8fcÃ¼\x18\x10Ãˆ\x9a\x98Ã™|Âº\x965|Â­ÃŽ\x00\x00\x1aÃ€/Ã€+Ã€\x11Ã€\x07Ã€\x13Ã€\x09Ã€\x14Ã€[0m" HTTPStatus.BAD_REQUEST -
138.68.174.63 - - [04/Aug/2024 19:17:15] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03_lP\x19Â²\x83')
138.68.174.63 - - [04/Aug/2024 19:17:15] "[35m[1m\x16\x03\x01\x00{\x01\x00\x00w\x03\x03_lP\x19Â²\x83\x1fÂ³Ã¯S\x08Ã˜l\x9bÃ›[0m" HTTPStatus.BAD_REQUEST -
138.68.174.63 - - [04/Aug/2024 19:17:15] "GET / HTTP/1.1" 200 -
205.210.31.140 - - [04/Aug/2024 20:37:58] "GET / HTTP/1.1" 200 -
118.194.251.141 - - [04/Aug/2024 20:56:02] "GET / HTTP/1.1" 200 -
198.235.24.16 - - [04/Aug/2024 21:03:09] "GET / HTTP/1.1" 200 -
205.210.31.38 - - [05/Aug/2024 02:39:40] "GET / HTTP/1.0" 200 -
120.56.43.10 - - [05/Aug/2024 03:54:12] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
52.167.144.54 - - [05/Aug/2024 03:56:36] "[33mGET /robots.txt HTTP/1.1[0m" 404 -
40.77.167.7 - - [05/Aug/2024 03:56:39] "GET / HTTP/1.1" 200 -
40.77.167.43 - - [05/Aug/2024 03:58:03] "[31m[1mGET /login HTTP/1.1[0m" 405 -
120.56.43.10 - - [05/Aug/2024 03:58:17] "GET / HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 03:58:20] "GET / HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 03:58:32] "[35m[1mPOST /login HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 705, in cmd_query
    self._cmysql.query(
_mysql_connector.MySQLInterfaceError: Lost connection to MySQL server during query

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1498, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1476, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/root/loginapp/app.py", line 25, in login
    cursor.execute("SELECT * FROM users WHERE username=%s AND password=%s", (username, password))
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/cursor_cext.py", line 357, in execute
    result = self._connection.cmd_query(
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/opentelemetry/context_propagation.py", line 97, in wrapper
    return method(cnx, *args, **kwargs)
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 713, in cmd_query
    raise get_mysql_exception(
mysql.connector.errors.OperationalError: 2013 (HY000): Lost connection to MySQL server during query
120.56.43.10 - - [05/Aug/2024 03:58:32] "GET /login?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 03:58:33] "GET /login?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 03:58:33] "GET /login?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 03:58:56] "[35m[1mPOST /login HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 705, in cmd_query
    self._cmysql.query(
_mysql_connector.MySQLInterfaceError: Lost connection to MySQL server during query

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1498, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1476, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/root/loginapp/app.py", line 25, in login
    cursor.execute("SELECT * FROM users WHERE username=%s AND password=%s", (username, password))
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/cursor_cext.py", line 357, in execute
    result = self._connection.cmd_query(
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/opentelemetry/context_propagation.py", line 97, in wrapper
    return method(cnx, *args, **kwargs)
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 713, in cmd_query
    raise get_mysql_exception(
mysql.connector.errors.OperationalError: 2013 (HY000): Lost connection to MySQL server during query
120.56.43.10 - - [05/Aug/2024 03:58:57] "[36mGET /login?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
120.56.43.10 - - [05/Aug/2024 03:58:57] "[36mGET /login?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
40.77.167.43 - - [05/Aug/2024 03:59:03] "[31m[1mGET /login HTTP/1.1[0m" 405 -
40.77.167.7 - - [05/Aug/2024 03:59:48] "GET / HTTP/1.1" 200 -
40.77.167.27 - - [05/Aug/2024 04:00:14] "[31m[1mGET /login HTTP/1.1[0m" 405 -
40.77.167.27 - - [05/Aug/2024 04:01:54] "[31m[1mGET /login HTTP/1.1[0m" 405 -
40.77.167.27 - - [05/Aug/2024 04:04:34] "[31m[1mGET /login HTTP/1.1[0m" 405 -
205.210.31.22 - - [05/Aug/2024 04:07:43] "GET / HTTP/1.0" 200 -
120.56.43.10 - - [05/Aug/2024 04:07:56] "GET / HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:08:19] "[35m[1mPOST /login HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 705, in cmd_query
    self._cmysql.query(
_mysql_connector.MySQLInterfaceError: Lost connection to MySQL server during query

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1498, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1476, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/root/loginapp/app.py", line 25, in login
    cursor.execute("SELECT * FROM users WHERE username=%s AND password=%s", (username, password))
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/cursor_cext.py", line 357, in execute
    result = self._connection.cmd_query(
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/opentelemetry/context_propagation.py", line 97, in wrapper
    return method(cnx, *args, **kwargs)
  File "/usr/local/lib64/python3.9/site-packages/mysql/connector/connection_cext.py", line 713, in cmd_query
    raise get_mysql_exception(
mysql.connector.errors.OperationalError: 2013 (HY000): Lost connection to MySQL server during query
120.56.43.10 - - [05/Aug/2024 04:08:20] "GET /login?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:08:20] "GET /login?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:08:20] "GET /login?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
40.77.167.27 - - [05/Aug/2024 04:10:15] "[31m[1mGET /login HTTP/1.1[0m" 405 -
87.236.176.76 - - [05/Aug/2024 04:13:15] "GET / HTTP/1.1" 200 -
 * Detected change in '/root/loginapp/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 433-626-050
 * Serving Flask app 'app'
 * Debug mode: on
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.31.92.215:5000
[33mPress CTRL+C to quit[0m
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 121-342-651
120.56.43.10 - - [05/Aug/2024 04:18:22] "GET / HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:18:28] "[32mPOST /login HTTP/1.1[0m" 302 -
120.56.43.10 - - [05/Aug/2024 04:18:28] "GET /videos HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:18:48] "GET /documents HTTP/1.1" 200 -
52.167.144.197 - - [05/Aug/2024 04:18:57] "[31m[1mGET /login HTTP/1.1[0m" 405 -
120.56.43.10 - - [05/Aug/2024 04:19:31] "GET / HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:19:46] "GET /register HTTP/1.1" 200 -
120.56.43.10 - - [05/Aug/2024 04:23:59] "[32mPOST /login HTTP/1.1[0m" 302 -
120.56.43.10 - - [05/Aug/2024 04:23:59] "GET /videos HTTP/1.1" 200 -
